src/components/membership/index.tsx

```diff
@@ -1,15 +1,174 @@
+    },
 }));
 
-export default function MembershipListPage() {
-    
+export default function MembershipListPage({ memberships }: MembershipListPageProps) {
+    const classes = useStyles();
+    const [openId, setOpenId] = useState<string | null>(null);
+    const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
+
+    const handlePopoverOpen = (event: React.MouseEvent<HTMLElement, MouseEvent>) => {
+      setAnchorEl(event.currentTarget);
+    };
+  
+    const handlePopoverClose = () => {
+      setAnchorEl(null);
+    };
+
+    const handleClickOpen = (id: string) => {
+        setOpenId(id);
+    };
+  
+    const handleClose = () => {
+        setOpenId(null);
+    };
+
+    const popoverOpen = Boolean(anchorEl);
+
     return (
-        <Box>
-            membership
+        <Box className={classes.wrapper}>
+            <List component="nav">
+                {
+                    memberships.map((membership, index) => {
+                        const id = `${index}-${membership.title}`;
+                        return (
+                            <span key={membership.title}>
+                                <ListItem className={classes.listItem} button divider onClick={() => handleClickOpen(id)}>
+                                    <WebsiteInfo title={membership.title} favicon={membership.favicon} />
+                                    <ListItemIcon>
+                                        <ChevronRightIcon />
+                                    </ListItemIcon>
+                                </ListItem> 
+                                <Dialog 
+                                    keepMounted
+                                    open={id === openId}
+                                    onClose={handleClose} 
+                                    classes={{ paper: classes.dialog }} 
+                                    style={{backgroundColor: 'transparent'}}
+                                    BackdropProps={{
+                                        classes: {
+                                            root: classes.backDrop,
+                                        }
+                                    }}
+                                >
+                                    <DialogTitle id={`${membership.title}-dialog-title`}>{membership.title}</DialogTitle>
+                                    <DialogContent>
+                                        {
+                                            membership.infos.map((i) => {
+                                                return (
+                                                    <Typography key={i.label} className={classes.text}>
+                                                        {i.label.toLowerCase()} : {i.data}
+                                                    </Typography>
+                                                )
+                                            })
+                                        }
```

Comment:
Appearance when the element of the list clicked (modal opened).
<img width="346" alt="스크린샷 2020-09-02 오후 5 51 00" src="https://user-images.githubusercontent.com/14817195/91960437-ded0b400-ed44-11ea-972e-0c447ab3c997.png">